mmvb_backend/case_synthesizer/generator.py

```diff
@@ -65,82 +194,12 @@ def generate_cases(quantity):
 
     for number in range(quantity):
         case_id = generate_id()
-        age = random.randint(MIN_AGE, MAX_AGE)
-        biological_sex = random.choice(BIOLOGICAL_SEXES)
-        meta_data = {
-            "description": f"Synthetic London-model case ({number+1}/{quantity})"
-        }
-
-        # Sample from conditions based on their probabilities
-        condition_probabilities = [
-            condition_info["probability"][biological_sex]
-            for condition_info in FIXTURES_DATA["conditions"]
-        ]
-
-        sampled_condition = random.choices(
-            FIXTURES_DATA["conditions"], condition_probabilities
-        )[0]
-
-        symptom_probabilities = {
-            symptom_id: probability
-            for condition_id, symptom_id, probability in FIXTURES_DATA[
-                "condition_symptom_probability"
-            ]
-            if condition_id == sampled_condition["id"]
+        case_data = generate_case_data()
+        metadata = {
+            "description": f"Synthetic Berlin-model case ({number+1}/{quantity})",
+            "caseCreator": "MMVB case synthesizer",
         }
-
-        if len(symptom_probabilities) <= 0:
-            raise SynthesisError(
-                "Something went wrong when generating symptom probabilities"
-            )
-
-        symptom_states = sample_symptoms(symptom_probabilities)
-
-        if len(symptom_states) <= 0:
-            raise SynthesisError(
-                "Something went wrong when sampling symptom states"
-            )
-
-        symptoms = [
-            combine_symptom_and_state(
-                symptom, symptom_states.get(symptom["id"])
-            )
-            for symptom in FIXTURES_DATA["symptoms"]
-            if symptom["id"] in symptom_states
-        ]
-
-        # Split a present symptom as presenting symptom
-        presenting_symptom = symptoms.pop(
-            [
-                index
-                for index, symptom in enumerate(symptoms)
-                if symptom["state"] == PRESENT
-            ][0]
-        )
-
-        case = Case(
-            id=case_id,
-            data={
-                "caseData": {
-                    "metaData": meta_data,
-                    "profileInformation": {
-                        "biologicalSex": biological_sex,
-                        "age": age,
-                    },
-                    "presentingComplaints": [presenting_symptom],
-                    "otherFeatures": symptoms,
-                },
-                "valuesToPredict": {
-                    "expectedTriageLevel": (
-                        sampled_condition["expected_triage_level"]
-                    ),
-                    "condition": {
-                        "id": sampled_condition["id"],
-                        "name": sampled_condition["name"],
-                    },
-                },
-            },
-        )
+        case = Case(id=case_id, data=case_data, metadata=metadata)
```

Comment:
According to the [proposed schema](https://github.com/FG-AI4H-TG-Symptom/fgai4h-tg-symptom-models-schemas/blob/master/schemas/berlin-model.schema.json#L36-L65) a case should have at least a name (also possible to have a description and case creator).
What would be the best approach here, adding a metadata field to the `Case` model? Adding multiple fields (`name`, `description`, `case_creator` to it? Or pushing the metada alongside the "data" contents within the `data` field? See comment above.

Reply:
Thanks @maccinza - good catch! Have changed this to add the metadata to the case alongside `caseData` and `valuesToPredict` as suggested (with just name and case_creator for now)